### PR TITLE
mgr/dashboard: fix API tests to be py3 compatible

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test docs

File: qa/tasks/vstart_runner.py [hunk 1]
```diff
@@ -940,7 +940,7 @@ def run_ceph_w(self, watch_channel=None):
         if watch_channel is not None:
             args.append("--watch-channel")
             args.append(watch_channel)
-        proc = self.controller.run(args=args, wait=False, stdout=BytesIO())
+        proc = self.controller.run(args=args, wait=False, stdout=StringIO())
```
Comment (lxbsz) [inline qa/tasks/vstart_runner.py:943]: @tchaikov 
The https://github.com/ceph/ceph/pull/34732/commits/dd4fc808a8a26ebbfae5b5e4bc3b6c99bf0f5d37 is also fixing it.

Comment (tchaikov) [reply qa/tasks/vstart_runner.py:943]: i am not sure what i am supposed to do? this commit has not been merged. neither can i find the PR containing it.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2020-04-26_09:25:11-rados-wip-kefu-testing-2020-04-26-1313-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2020-04-26_13:55:50-rados-wip-kefu-testing-2020-04-26-1314-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2020-04-27_01:47:10-rados-wip-kefu-testing-2020-04-26-1314-distro-basic-smithi/

File: qa/tasks/vstart_runner.py (hunk 1 above)
Comment (lxbsz) [reply qa/tasks/vstart_runner.py:943]: Sorry, I misreaded it. Thanks.

**Comment by alfonsomthd** [comment]
@tchaikov Should this backported to octopus?

**Comment by tchaikov** [comment]
> @tchaikov Should this backported to octopus?

yes, it's included in #34171 .

**Pull request merged**